src/site/content/en/progressive-web-apps/customize-install/index.md

```diff
@@ -30,68 +20,101 @@ it's possible to provide your own custom install flow, directly within your app.
   </figcaption>
 </figure>
 
-When considering whether to promote install, it's best to think about how
-users typically use your PWA.  For example, if there's a set of users who
-use your PWA multiple times in a week, these users might benefit from the
-added convenience of launching your app from a smartphone homescreen or
-from the Start menu in a desktop operating system.  Some productivity and
-entertainment applications also benefit from the extra screen real-estate
-created by removing the browser toolbars from the window in installed
-`standalone` or `minimal-ui` modes.
-
-<div class="w-clearfix"></div>
-
-## Promoting installation {: #promote-installation }
+Installability is one of the biggest advantages of a Progressive Web App.
+Installing your PWA can make it easier for users to find and use. Once
+installed, it looks and behaves like every other installed app that the user
+has.
 
-To indicate your Progressive Web App is installable, and to provide a custom
-in-app install flow:
+But, some users don't realize that they can install your PWA, so it can be
+helpful to provide an in-app experience to promote and enable installation of
+your PWA.
 
-1. Listen for the `beforeinstallprompt` event.
-2. Save the `beforeinstallprompt` event, so it can be used to trigger the
-   install flow later.
-3. Alert the user that your PWA is installable, and provide a button or other
-   element to start the in-app installation flow.
+<div id="promote-installation" class="w-clearfix"></div>
+
+## PWA Install Flow {: #install-flow }
+
+Let's take a look at the PWA install flow for Chrome and Edge, how you can
```

Comment:
```suggestion
Consider the PWA install flow for Chrome and Edge, how you can
```

Reply:
* Omit needless words (Shrunk and White)
* Avoid forms of 'we' (Our own handbook)